web/src/routes/(user)/albums/[albumId]/+page.svelte

```diff
@@ -82,6 +82,7 @@
 
   let album = data.album;
   let description = album.description;
```

Comment:
Looke like these variables don't account for `data` to change and I think that's the root cause for the issues you're seeing with not re-rendering. Would be preferable if it can be fixed by making all these variables reactive, instead of the hacky solution in `afterNavigate`.

Reply:
I don't think we should use reactive variables for description. When confirming the album description change, we compare the original album description with the new one. 

However, I forgot to re-initialize the description when navigating from one album to another (you can see on the video that the album description does not change, the description of the second album is "second album")

Reply:
After taking another look, there are many more (reactivity) issues in this component. Just making these reactive variables won't work indeed, but I still believe it's better to fix these issues instead of coming up with a workaround. Maybe if we create new components for album title/description and do a little refactoring we can avoid the workaround?

Reply:
I think this would be a better approach as well.

Reply:
I don't know if using components would help. From my understanding, this is more of a sveltekit issue with variables not being reset when the slug changes.

Reply:
This isn't a svelte kit issue afaik. If you navigate to the same route with a different slug the component won't remount, which makes sense. To support these types of changes the data in the component needs to be reactive to the route param change.